src/TextField/TextField.js

```diff
@@ -73,6 +76,15 @@ export default class TextField extends Component {
      * The CSS class name of the root element.
      */
     className: PropTypes.string,
+    /**
+     * Whether the label should be displayed in an error state
+     */
+    error: PropTypes.bool,
```

Comment:
Good API decision, looking at the docs, the error state has an effect on all the children:
E.g:
OK:
![capture d ecran 2016-11-22 a 20 04 19](https://cloud.githubusercontent.com/assets/3165635/20537812/e3548e0c-b0ee-11e6-81d5-546687cd9d12.png)

KO:
![capture d ecran 2016-11-22 a 20 04 39](https://cloud.githubusercontent.com/assets/3165635/20537819/eda30082-b0ee-11e6-87a3-3a4157c32057.png)